### PR TITLE
Allow WC's NAT IPs in security policy

### PR DESCRIPTION
**Pull request by mnitchev**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,112 +3,91 @@ module github.com/giantswarm/capg-firewall-rule-operator
...
-	k8s.io/apimachinery v0.23.6
-	k8s.io/client-go v0.23.6
-	sigs.k8s.io/cluster-api v1.0.5
-	sigs.k8s.io/cluster-api-provider-gcp v1.0.2
-	sigs.k8s.io/controller-runtime v0.11.2
+	google.golang.org/api v0.94.0
+	google.golang.org/genproto v0.0.0-20220829175752-36a9c930ecbf
+	k8s.io/api v0.25.0
+	k8s.io/apimachinery v0.25.0
+	k8s.io/client-go v0.25.0
```
Comment (fiunchinho) [inline go.mod:20]: This may be too modern? We are running 1.22 in the MCs

Comment (mnitchev) [reply go.mod:20]: They **should** be backwards compatible. I'd say we leave it like that and if it fails on a test installation we can revert to `v0.22.0`

Comment (mnitchev) [reply go.mod:20]: Unless there's something specific you think is going to break between these versions

Comment (mnitchev) [reply go.mod:20]: Tried it out with the image generated from CI and it seems to work fine

**Review by zewolfe — APPROVED** [review]
LGTM

**Comment by mnitchev** [comment]
Part of giantswarm/roadmap#1293

**Pull request merged**